packages/frontend/src/components/composer/ComposerMessageInput.tsx

```diff
@@ -124,17 +123,14 @@ export default class ComposerMessageInput extends React.Component<
       this.resizeTextareaAndComposer()
       if (prevState.text !== this.state.text) {
         if (!this.state.loadingDraft) {
-          this.saveDraft()
+          this.saveDraft(this.state.text, prevState.chatId)
         }
       }
     }
   }
 
   onChange(e: React.ChangeEvent<HTMLTextAreaElement>) {
     this.setState({ text: e.target.value /*error: false*/ })
-    if (!this.state.loadingDraft) {
-      this.saveDraft()
-    }
```

Comment:
Why is this removed? Is it just a refactor?

Reply:
These lines exist also in onComponentDidUpdate what doesn't make sense to me, since switching to another chat also triggers onComponentDidUpdate so in the end a fresh loaded draft is immediately saved again. So why not remove them there and leave them here?

Reply:
Makes sense.
But the question is if it's directly related to what this commit is doing.
Consider this a nitpick then.